### PR TITLE
Account Page Left Bar UI created

### PR DESCRIPTION
**Pull request by ejaz**

### CODE REVIEW

File: theme/layout/theme.liquid [hunk 1]
```diff
@@ -112,7 +112,7 @@ pintrk('page');
 </head>
 
 <body id="{{ langify_title | handle }}"
-  class=" {% if customer %}customer-logged-in {% endif %}template-{{ template | replace: '.', ' ' | truncatewords: 1, '' | handle }}">
+  class="codiTheme {% if customer %}customer-logged-in {% endif %}template-{{ template | replace: '.', ' ' | truncatewords: 1, '' | handle }}">
```
Comment (PygmalionPolymorph) [inline theme/layout/theme.liquid]: What does 'codiTheme' mean?

File: theme/templates/customers/account.liquid [hunk 2]
```diff
@@ -55,23 +144,19 @@
...
+        {{ customer.default_address | format_address }}
+        <p><a href="/account/addresses" class="btn btn--small">{% comment %}ly_i18n_replace_for_[ 'customer.account.view_addresses' | t ]_begin{% endcomment %}{% capture ly_template %}{{ 'customer.account.view_addresses' | t }}{% endcapture %}{% include 'ly-static-string' with '15088081' %}{% if ly_translation %}{% assign escape_content = true %}{% assign ly_template = ly_translation %}{% else %}{% assign escape_content = false %}{% endif %}{% if escape_content %}{{ ly_template | escape}}{% else %}{{ ly_template }}{% endif %}{% comment %}ly_i18n_replace_end{% endcomment %} ({{ customer.addresses_count }})</a></p>
+        <!-- Begin ReCharge code -->
+        <p id="rc_link_container">
+          <a href="/tools/recurring/login">Manage Subscriptions</a>
+        </p>
+        <!-- End ReCharge code -->
```
Comment (PygmalionPolymorph) [inline theme/templates/customers/account.liquid]: This ReCharge Block can be removed

File: theme/assets/account.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	transition: 0.3s all ease-in-out;
+}
+
+
+
+.twoCols .rightArea {
+  max-width: calc(100% - 432px);
+}
+
+@media only screen and (max-width: 800px) {
```
Comment (PygmalionPolymorph) [inline theme/assets/account.css]: Mobile should be up to 768 px

File: theme/templates/customers/account.liquid [hunk 4]
```diff
@@ -82,3 +167,4 @@
   <div class="logout">
 < {% comment %}ly_i18n_replace_for_[ 'layout.customer.log_out' | t | customer_logout_link ]_begin{% endcomment %}{% capture ly_template %}{{ 'layout.customer.log_out' | t | customer_logout_link }}{% endcapture %}{% include 'ly-static-string' with '15088082' %}{% if ly_translation %}{% assign escape_content = true %}{% assign ly_template = ly_translation %}{% else %}{% assign escape_content = false %}{% endif %}{% if escape_content %}{{ ly_template | escape | customer_logout_link }}{% else %}{{ ly_template }}{% endif %}{% comment %}ly_i18n_replace_end{% endcomment %}
   </div>
+<script src="{{ 'account.js' | asset_url }}"></script>
```
Comment (PygmalionPolymorph) [inline theme/templates/customers/account.liquid]: account.js is empty! Is there any logic to change the tabs? I can't see it

**Comment by ejaz** [comment]
Would Create New pull request.

**Pull request closed without merging**